### DISCUSSION TITLE
[Bug]: https urls not recognised/rendered

### DISCUSSION THREAD

**Discussion by PSG2017** [RafaelGB/obsidian-db-folder#773] [Q&A]
### What happened?

External urls commencing www are recognised when entered into a text field, provided they commence www. They render as a link that can be clicked to go to the website referred to. But addresses commencing with https render as the word Website (which is the name of the column). When I click to edit in the cell concerned I see the full url (starting with https). When I edit that into www instead of https then the text renders as an url (commencing www).

### What platform were you using?

Desktop

### Version of the plugin

3.2.4

### Are you using the latest version of the plugin in your Obsidian vault?

- [X] I have verified that I am on the latest version

### Are you check if there is a similar issue?

- [X] I have verified that there are not similar issues

**Comment by RafaelGB** [comment]
Hello @PSG2017 ,
Disable this option to render the URL on column settings

<img width="554" alt="Screenshot 2023-01-23 at 15 23 09" src="https://user-images.githubusercontent.com/11924043/214063343-b8a2477f-4050-4ef5-b759-de9035e0e96b.png">


With some links it renders and embed web page but its not working for all of them

**Reply by PSG2017** [reply]
Thanks. Just seen this and have tried it out and it certainly works on one link I was having this issue with. Have not yet reviewed all as I had converted them to www until this issue was resolved. I will begin converting them back and will report if there are any further issues. Great plugin thanks.
